src/porepy/models/contact_mechanics_biot_model.py

```diff
@@ -52,11 +52,7 @@ class ContactMechanicsBiot(pp.ContactMechanics):
     adjustment needed is to specify the method create_grid().
 
     Attributes:
-        time (float): Current time.
-        time_step (float): Size of an individual time step
-        time_index (int): Index of current time step. Used/updated in
-            run_time_dependent_model.
-        end_time (float): Time at which the simulation should stop.
+        tsc: Time-stepping control manager.
```

Comment:
Regarding documentation of attributes (though there was no discussion so far after my proposal), should we start commenting them in separate docstrings in the constructor?

Reply:
Yes, I didn't do it since it wasn't settled. But it makes sense to agree on this so new code can already be properly documented.